style.css

```diff
@@ -62,9 +466,260 @@
+  .about-info {
+    background-color: white;
+    border: 2px solid var(--colorGrey);
+    width: 50%;
+  }
+
+  .desktop-header {
+    display: block;
+  }
+
+  .desktop-secondary-nav {
+    background-color: white;
+    height: 90px;
+    position: fixed;
+    width: 100%;
+    top: 40px;
+    -webkit-box-shadow: 0 19px 20px -9px rgba(0, 0, 0, 0.32);
+    box-shadow: 0 19px 20px -9px rgba(0, 0, 0, 0.32);
+    z-index: 1;
+  }
+
+  .desktop-top-nav {
+    background-color: var(--colorDeepBrown);
+    height: 40px;
+    position: fixed;
+    width: 100%;
+    top: 0;
+    z-index: 1;
+  }
+
+  .home-grey-background {
+    display: none;
+  }
+
+  .desktop-home-background {
+    background-image: url("images/home-background.jpg");
+    background-repeat: repeat;
+    height: 100vh;
+    width: 100%;
+  }
+
+  .desktop-overlay {
+    height: 100vh;
+    background: rgba(255, 255, 255, 0.9);
+  }
+
+  .feature-speaker-list-parent {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: repeat(3, 1fr);
+    grid-column-gap: 46px;
+    grid-row-gap: 19px;
+    list-style: none;
+    padding: 0;
+    margin-left: 20%;
+    margin-right: 20%;
+  }
+
+  .footer-content {
+    display: flex;
+    justify-content: space-around;
+    margin-top: 50px;
+    margin-bottom: 50px;
+  }
+
+  .footer-content > p {
+    font-family: var(--fontFamilyLato);
+  }
+
+  .conf-info {
+    position: absolute;
+    top: 150px;
+    margin-left: 15%;
+  }
+
+  .contact-us-info {
+    font-family: var(--fontFamilyLato);
+    width: 40%;
+    text-align: center;
+  }
+
+  .hello-text {
+    margin-top: 10%;
+  }
+
+  .hidden-feature-list {
+    display: block;
+  }
+
+  .hr-long {
+    background-color: var(--colorOrange);
+    height: 1px;
+    width: 35px;
+    border-width: 0;
+  }
+
+  .join-button-btn {
+    display: none;
+  }
+
+  .link {
+    text-decoration: none;
+    color: var(--colorDeepBrown);
+  }
+
+  .logo-desc {
+    font-family: var(--fontFamilyLato);
+    width: 40%;
+    text-align: center;
+  }
+
+  .logo-small {
+    width: 15%;
+    height: auto;
+    margin-right: 20px;
+  }
+
+  .main-program {
+    display: flex;
+    flex-direction: column;
+    color: white;
+  }
+
+  .main-title {
+    background-image: url("./images/text-background.jpg");
+    background-repeat: repeat;
+    max-height: max-content;
+    color: var(--colorOrange);
+    font-family: var(--fontFamilyCocogoose);
+    line-height: 90%;
+    font-size: 40px;
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+    font-weight: bold;
+  }
+
+  .more-class {
+    display: none;
+  }
+
+  .partner {
+    height: 340px;
+  }
+
+  .partner-logos > li {
+    width: 20%;
+    margin-left: 20px;
+    margin-right: 20px;
+  }
+
+  .partner-logos {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    list-style: none;
+    padding: 0;
+    width: 30%;
+    height: auto;
+  }
```

Comment:
- [ ] Kindly remove this `width: 30%;`, and add `align-items: center;`.
- [ ] And on your `.partner-logo` class, on line 181, Make the `width` equal 60% instead of 100%. It will make your partners section much better.

Like
```
.partner-logo {
    width: 60%;
    height: auto;
}
```

Your section 
![image](https://user-images.githubusercontent.com/35535130/158817552-c181721b-b9dd-44f6-8b24-9f0a5bb69a96.png)

Original section
![image](https://user-images.githubusercontent.com/35535130/158817661-71363960-cbb6-4236-96d2-e117373e9432.png)